docker-compose.yml

```diff
@@ -7,6 +7,7 @@ services:
     privileged: true
     devices:
       - /dev/bus/usb:/dev/bus/usb
+      - /dev/apex_0:/dev/apex_0
```

Comment:
Does this prevent the container from starting on device types where this is not present? I will have to test on NUC and Raspberry Pi unless you already have?